services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    return list(Ticket.objects.filter(
+        movie_session_id=movie_session_id).values("row", "seat"))
```

Comment:
Reformat this code and place the closing parenthesis for `filter` to a new line to match the opening parenthesis that does not have arguments on the same line.